db/versions/0033.yml

```diff
@@ -113,6 +117,7 @@ methods:
         end if;
       end
   queues_entities_scan:
+    deprecated: true
```

Comment:
I was expecting to see `deprecated-by` rather than `deprecated` per https://github.com/taskcluster/taskcluster/pull/3331#discussion_r465276574. Was a commit not pushed for this?

Reply:
I changed the whole approach, so the DB version file no longer has a TC version in it.